packages/tupaia-web/src/features/Visuals/View/MultiPhotograph/MultiPhotographPreview.tsx

```diff
@@ -31,15 +31,51 @@ const Thumbnail = styled.div<{
   }
 `;
 
+const ExportWrapper = styled.div``;
+
+const Image = styled.img`
+  width: 48%;
+  height: auto;
+  border: 1px solid #000;
+  background-image: url(${({ src }) => src});
+  background-size: cover;
+  background-repeat: no-repeat;
+  margin-bottom: 10px;
+
+  &:nth-child(n) {
+    margin-right: 1%;
+  }
+  &:nth-child(2n) {
+    margin-left: 1%;
+  }
+`;
+
 interface MultiPhotographPreviewProps {
   report: ViewReport;
   config: ViewConfig;
+  isExport?: boolean;
 }
 
 export const MultiPhotographPreview = ({
   report: { data = [] },
   config,
+  isExport,
 }: MultiPhotographPreviewProps) => {
+  if (isExport) {
```

Comment:
I would almost like this to be a separate component, since it's quite different to where the preview appears and has it's own logic